### PR TITLE
Remove NullHandler from logger since it filters messages

### PR DESCRIPTION
**Pull request by wkentaro**
Close https://github.com/mikedh/trimesh/issues/396

It filters every warnings and errors.

I'm not sure if how you decided adding NullHandler and if it is right answer removing it.

### CODE REVIEW

**Comment by wkentaro** [comment]
Now it shows like below:
```
$ python
>>> import trimesh
>>> trimesh.load('models/fuze.obj')
failed to load texture: material_0
Traceback (most recent call last):
  File "/home/wkentaro/Documents/trimesh/trimesh/exchange/wavefront.py", line 171, in append_mesh
    image = PIL.Image.open(
NameError: name 'PIL' is not defined
<trimesh.base.Trimesh object at 0x7fbeca8c9358>
```

File: trimesh/collision.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 try:
     import fcl  # pip install python-fcl
 except BaseException:
-    log.warning('No FCL -- collision checking will not work')
```
Comment (wkentaro) [inline trimesh/collision.py]: This is because it's too noisy, and make it consistent to the import of `PIL.Image` or other libraries.

**Comment by mikedh** [comment]
Oh nice, thanks for tracking this one down! I think the `NullHandler` was probably added as the [examples in the Python docs do it](https://docs.python.org/2/howto/logging.html). Makes sense to get rid of it though.  

Thanks for the PR!

**Comment by mikedh** [comment]
Also, it's pretty sweet that CI caught that `flake8` warning now :smile:

**Comment by wkentaro** [comment]
Oops :sweat_smile: 

```
% flake8 trimesh/
trimesh/collision.py:5:1: F401 '.constants.log' imported but unused
```

**Pull request merged**